android/java/org/chromium/chrome/browser/util/TabUtils.java

```diff
@@ -175,50 +187,63 @@ private static void openNewTab(BraveActivity braveActivity, boolean isIncognito)
+                    && braveActivity.getActivityTab() != null) {
+                braveActivity.getTabModelSelector().openNewTab(new LoadUrlParams(url),
+                        TabLaunchType.FROM_LONGPRESS_BACKGROUND_IN_GROUP,
+                        braveActivity.getActivityTab(), isIncognito);
+            }
+        } catch (ActivityNotFoundException e) {
         }
     }
 
     public static void openUrlInSameTab(String url) {
-        BraveActivity braveActivity = BraveActivity.getBraveActivity();
-        if (braveActivity != null && braveActivity.getActivityTab() != null) {
-            LoadUrlParams loadUrlParams = new LoadUrlParams(url);
-            braveActivity.getActivityTab().loadUrl(loadUrlParams);
+        try {
+            BraveActivity braveActivity = BraveActivity.getBraveActivity();
+            if (braveActivity != null && braveActivity.getActivityTab() != null) {
+                LoadUrlParams loadUrlParams = new LoadUrlParams(url);
+                braveActivity.getActivityTab().loadUrl(loadUrlParams);
+            }
+        } catch (ActivityNotFoundException e) {
         }
     }
 
     public static void enableRewardsButton() {
-        BraveActivity braveActivity = BraveActivity.getBraveActivity();
-        if (braveActivity == null || braveActivity.getToolbarManager() == null) {
-            return;
-        }
-        View toolbarView = braveActivity.findViewById(R.id.toolbar);
-        if (toolbarView == null) {
-            return;
-        }
-        FrameLayout rewardsLayout = toolbarView.findViewById(R.id.brave_rewards_button_layout);
-        if (rewardsLayout == null) {
-            return;
+        try {
+            BraveActivity braveActivity = BraveActivity.getBraveActivity();
+            if (braveActivity == null || braveActivity.getToolbarManager() == null) {
+                return;
+            }
+            View toolbarView = braveActivity.findViewById(R.id.toolbar);
+            if (toolbarView == null) {
+                return;
+            }
+            FrameLayout rewardsLayout = toolbarView.findViewById(R.id.brave_rewards_button_layout);
+            if (rewardsLayout == null) {
+                return;
+            }
+            rewardsLayout.setVisibility(View.VISIBLE);
+        } catch (ActivityNotFoundException e) {
         }
-        rewardsLayout.setVisibility(View.VISIBLE);
     }
 
     public static void bringChromeTabbedActivityToTheTop(Activity activity) {
-        Intent braveActivityIntent = new Intent(activity, ChromeTabbedActivity.class);
-        braveActivityIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
-        activity.startActivity(braveActivityIntent);
+        try {
+            Intent braveActivityIntent = new Intent(activity, ChromeTabbedActivity.class);
+            braveActivityIntent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP);
+            activity.startActivity(braveActivityIntent);
+        } catch (ActivityNotFoundException e) {
```

Comment:
why would we add try catch in that place?

Reply:
updated